content/learn/book/getting-started/_index.md

```diff
@@ -2,6 +2,7 @@
 title = "Getting Started"
 template = "docs-section.html"
 insert_anchor_links = "right"
+redirect_to = "/learn/quick-start/introduction"
```

Comment:
A last second thought; adding a comment above this line to explain why this is here and that you need to temporarily remove it to see the page whilst working on it is probably a good idea.

Reply:
@TrialDragon Done